infrastructure/network/netadapter/server/grpcserver/protowire/messages.proto

```diff
@@ -121,6 +121,8 @@ message KaspadMessage {
     NotifyVirtualDaaScoreChangedRequestMessage notifyVirtualDaaScoreChangedRequest = 1074;
     NotifyVirtualDaaScoreChangedResponseMessage notifyVirtualDaaScoreChangedResponse = 1075;
     VirtualDaaScoreChangedNotificationMessage virtualDaaScoreChangedNotification = 1076;
+    GetBalanceByAddressRequestMessage getBalanceByAddressRequestMessage = 1077;
+    GetBalanceByAddressResponseMessage getBalanceByAddressResponseMessage = 1078;
```

Comment:
For consistency's sake, please remove the "Message" suffix from the variable names

Reply:
Fixed